src/instructions/Instructions.test.jsx

```diff
@@ -502,14 +502,15 @@ describe('SequenceExamWrapper', () => {
     expect(screen.getByTestId('retry-exam-button')).toHaveTextContent('Retry my exam');
   });
 
-  it('Shows submitted practice exam instructions if exam is onboarding and attempt status is submitted', () => {
+  it('Shows submitted practice exam instructions if exam is onboarding and attempt status is submitted on legacy LTI exams', () => {
     store.getState = () => ({
       specialExams: Factory.build('specialExams', {
         activeAttempt: {},
         exam: Factory.build('exam', {
           is_proctored: true,
           type: ExamType.PRACTICE,
           attempt: Factory.build('attempt', {
+            use_legacy_attempt_api: true,
             attempt_status: ExamStatus.SUBMITTED,
```

Comment:
Now if a Proctored exam with an LTI provider is submitted no longer shows the "Retry my exam" button. That's the reason for this update.

Reply:
NIT: we should remove LTI from this since LTI implies the new (not legacy) exam system